develop/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM opxhub/gbp
+
+# add extra tools required for development
+RUN apt-get -qq update && apt-get -qq upgrade -y && \
+    apt-get -qq install -y vim
```

Comment:
Need to clean up after installing. See the other Dockerfile.

https://github.com/famleebob/gbp-docker/blob/AddDevDocker/build/Dockerfile#L6